charts/lagoon-core/templates/broker.configmap.tlsconf.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.broker.tls.enabled }}
+---
+apiVersion: v1
+kind: Secret
```

Comment:
The `kind` doesn't match the filename, can it be renamed or move the data to `broker.secret.yaml`?